src/StripeTests/Infrastructure/ParameterBuilderTest.cs

```diff
@@ -55,6 +55,8 @@ public void EncodesParameters()
                     { "A", "Value-A" },
                     { "B", "Value-B" },
                 },
+                ListOfDoubles = new List<double> {1.5, 2.6, 3.7},
+                ArrayOfInts = new int[] {7, 8, 9},
```

Comment:
Totally random but thought about this during an API review. How do you *add* the number 10 to the existing list of 7,8,9 in dotnet? Usually in curl you would do `array_of_ints[][10]` is that still doable/allowed?

Reply:
For lists, you'd simply do `testObject.ListOfDoubles.Add(4.8)`.

For arrays, that's not possible as arrays are fixed size in C#. Initializing with `[]` is just syntactic sugar for asking the compiler to figure out the size based on the number of elements provided in the initializer (in this case 3).

(We should rarely be using arrays.)

Reply:
I'm not sure I follow what you mean here. Can you give me an example of how I would add a fourth element to an _existing_ list? This can't be done on the list itself it has to be done on a new options. Am I expected to copy all 4 elements in the list again before calling the update method?